packages/rum/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "elastic-apm-js-base",
   "version": "3.0.0",
   "description": "Elastic APM JavaScript agent",
-  "main": "dist/bundles/elastic-apm-js-base.umd.js",
+  "main": "src/index.js",
```

Comment:
@vigneshshanmugam, We should at least consider this a breaking change.

 Let's wait until the next major version and then we can merge this one.

Reply:
Sure, I am good with that.